locales/puppet.pot

```diff
@@ -4015,11 +4015,11 @@ msgstr ""
 msgid "WEBrick server is not listening"
 msgstr ""
 
-#: ../lib/puppet/network/http/webrick.rb:99
+#: ../lib/puppet/network/http/webrick.rb:98
```

Comment:
I think you'll want to accept "ours" versions of the pot localization file, and the po file above.